cookbook-connector/src/main/java/org/mule/modules/cookbook/config/BasicConfig.java

```diff
@@ -20,12 +20,12 @@
 import com.cookbook.tutorial.service.InvalidCredentialsException;
 
 /**
- * Connection Management handles login/logout for the user as well as reconnections.
+ * Connection Management handles login/logout for the user as well as reconnection's.
  *
  * @author MuleSoft, Inc.
  */
-@ConnectionManagement(friendlyName = "Configuration config")
-public class ConnectionManagementConfig extends ConnectorConfig {
+@ConnectionManagement(friendlyName = "Basic Configuration")
+public class BasicConfig extends AbstractConfig {
```

Comment:
It would makes sense to name the "BasicConfig" to "Config" . BasicConfig gives a sense that you have an AdvancedConfig as well, which is not true in this case.